[init : remove] Removing all files from /workspace/source ...

[clone : clone] + '[' false '=' true ]
[clone : clone] + '[' false '=' true ]
[clone : clone] + '[' false '=' true ]
[clone : clone] + CHECKOUT_DIR=/workspace/output/
[clone : clone] + '[' true '=' true ]
[clone : clone] + cleandir
[clone : clone] + '[' -d /workspace/output/ ]
[clone : clone] + rm -rf '/workspace/output//*'
[clone : clone] + rm -rf '/workspace/output//.[!.]*'
[clone : clone] + rm -rf '/workspace/output//..?*'
[clone : clone] + test -z 
[clone : clone] + test -z 
[clone : clone] + test -z 
[clone : clone] + git config --global --add safe.directory /workspace/output
[clone : clone] + /ko-app/git-init '-url=https://github.com/Ferddypaddy/devops-capstone-project.git' '-revision=cd-pipeline' '-refspec=' '-path=/workspace/output/' '-sslVerify=true' '-submodules=true' '-depth=1' '-sparseCheckoutDirectories='
[clone : clone] {"level":"info","ts":1675047239.3519278,"caller":"git/git.go:176","msg":"Successfully cloned https://github.com/Ferddypaddy/devops-capstone-project.git @ 99e8423c16c05e3439aa133fc569c3f1fcedd1f2 (grafted, HEAD, origin/cd-pipeline) in path /workspace/output/"}
[clone : clone] {"level":"info","ts":1675047239.4110446,"caller":"git/git.go:215","msg":"Successfully initialized and updated submodules in path /workspace/output/"}
[clone : clone] + cd /workspace/output/
[clone : clone] + git rev-parse HEAD
[clone : clone] + RESULT_SHA=99e8423c16c05e3439aa133fc569c3f1fcedd1f2
[clone : clone] + EXIT_CODE=0
[clone : clone] + '[' 0 '!=' 0 ]
[clone : clone] + git log -1 '--pretty=%ct'
[clone : clone] + RESULT_COMMITTER_DATE=1675047055
[clone : clone] + printf '%s' 1675047055
[clone : clone] + printf '%s' 99e8423c16c05e3439aa133fc569c3f1fcedd1f2
[clone : clone] + printf '%s' https://github.com/Ferddypaddy/devops-capstone-project.git

[tests : nosetests] ***** Installing dependencies *****
[tests : nosetests] Requirement already satisfied: pip in /usr/local/lib/python3.9/site-packages (22.0.4)
[tests : nosetests] Collecting pip
[tests : nosetests]   Downloading pip-22.3.1-py3-none-any.whl (2.1 MB)
[tests : nosetests]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 2.1/2.1 MB 39.5 MB/s eta 0:00:00
[tests : nosetests] Requirement already satisfied: wheel in /usr/local/lib/python3.9/site-packages (0.38.4)
[tests : nosetests] Installing collected packages: pip
[tests : nosetests]   Attempting uninstall: pip
[tests : nosetests]     Found existing installation: pip 22.0.4
[tests : nosetests]     Uninstalling pip-22.0.4:
[tests : nosetests]       Successfully uninstalled pip-22.0.4
[tests : nosetests] Successfully installed pip-22.3.1
[tests : nosetests] WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
[tests : nosetests] WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
[tests : nosetests] ***** Running nosetests with: -v --with-spec --spec-color
[tests : nosetests] nose.config: INFO: Ignoring files matching ['^\\.', '^_', '^setup\\.py$']
[tests : nosetests] nose.plugins.cover: INFO: Coverage report will include only packages: ['service']
[tests : nosetests] 
[tests : nosetests] Unloadable or unexecutable test.
[tests : nosetests] 
[tests : nosetests]     A Failure case is placed in a test suite to indicate the presence of a
[tests : nosetests]     test that could not be loaded or executed. A common example is a test
[tests : nosetests]     module that fails to import.
[tests : nosetests]     
[tests : nosetests]     
[tests : nosetests] [1;31m- runTest (ERROR)[1;0m
[tests : nosetests] [1;31m- runTest (ERROR)[1;0m
[tests : nosetests] [1;31m- runTest (ERROR)[1;0m
[tests : nosetests] [1;31m- runTest (ERROR)[1;0m
[tests : nosetests] 
[tests : nosetests] ======================================================================
[tests : nosetests] ERROR: Failure: AttributeError (module 'sqlalchemy' has no attribute '__all__')
[tests : nosetests] ----------------------------------------------------------------------
[tests : nosetests] Traceback (most recent call last):
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/failure.py", line 39, in runTest
[tests : nosetests]     raise self.exc_val.with_traceback(self.tb)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/loader.py", line 417, in loadTestsFromName
[tests : nosetests]     module = self.importer.importFromPath(
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/importer.py", line 47, in importFromPath
[tests : nosetests]     return self.importFromDir(dir_path, fqname)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/importer.py", line 94, in importFromDir
[tests : nosetests]     mod = load_module(part_fqname, fh, filename, desc)
[tests : nosetests]   File "/usr/local/lib/python3.9/imp.py", line 244, in load_module
[tests : nosetests]     return load_package(name, filename)
[tests : nosetests]   File "/usr/local/lib/python3.9/imp.py", line 216, in load_package
[tests : nosetests]     return _load(spec)
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 711, in _load
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
[tests : nosetests]   File "<frozen importlib._bootstrap_external>", line 850, in exec_module
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
[tests : nosetests]   File "/workspace/source/service/__init__.py", line 22, in <module>
[tests : nosetests]     from service import routes, models  # noqa: F401 E402
[tests : nosetests]   File "/workspace/source/service/routes.py", line 8, in <module>
[tests : nosetests]     from service.models import Account
[tests : nosetests]   File "/workspace/source/service/models.py", line 13, in <module>
[tests : nosetests]     db = SQLAlchemy()
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 758, in __init__
[tests : nosetests]     _include_sqlalchemy(self, query_class)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 112, in _include_sqlalchemy
[tests : nosetests]     for key in module.__all__:
[tests : nosetests] AttributeError: module 'sqlalchemy' has no attribute '__all__'
[tests : nosetests] 
[tests : nosetests] ======================================================================
[tests : nosetests] ERROR: Failure: AttributeError (module 'sqlalchemy' has no attribute '__all__')
[tests : nosetests] ----------------------------------------------------------------------
[tests : nosetests] Traceback (most recent call last):
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/failure.py", line 39, in runTest
[tests : nosetests]     raise self.exc_val.with_traceback(self.tb)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/loader.py", line 417, in loadTestsFromName
[tests : nosetests]     module = self.importer.importFromPath(
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/importer.py", line 47, in importFromPath
[tests : nosetests]     return self.importFromDir(dir_path, fqname)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/importer.py", line 94, in importFromDir
[tests : nosetests]     mod = load_module(part_fqname, fh, filename, desc)
[tests : nosetests]   File "/usr/local/lib/python3.9/imp.py", line 234, in load_module
[tests : nosetests]     return load_source(name, filename, file)
[tests : nosetests]   File "/usr/local/lib/python3.9/imp.py", line 171, in load_source
[tests : nosetests]     module = _load(spec)
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 711, in _load
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
[tests : nosetests]   File "<frozen importlib._bootstrap_external>", line 850, in exec_module
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
[tests : nosetests]   File "/workspace/source/tests/test_cli_commands.py", line 8, in <module>
[tests : nosetests]     from service.common.cli_commands import db_create
[tests : nosetests]   File "/workspace/source/service/common/cli_commands.py", line 4, in <module>
[tests : nosetests]     from service import app
[tests : nosetests]   File "/workspace/source/service/__init__.py", line 22, in <module>
[tests : nosetests]     from service import routes, models  # noqa: F401 E402
[tests : nosetests]   File "/workspace/source/service/routes.py", line 8, in <module>
[tests : nosetests]     from service.models import Account
[tests : nosetests]   File "/workspace/source/service/models.py", line 13, in <module>
[tests : nosetests]     db = SQLAlchemy()
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 758, in __init__
[tests : nosetests]     _include_sqlalchemy(self, query_class)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 112, in _include_sqlalchemy
[tests : nosetests]     for key in module.__all__:
[tests : nosetests] AttributeError: module 'sqlalchemy' has no attribute '__all__'
[tests : nosetests] -------------------- >> begin captured logging << --------------------
[tests : nosetests] faker.factory: DEBUG: Not in REPL -> leaving logger event level as is.
[tests : nosetests] --------------------- >> end captured logging << ---------------------
[tests : nosetests] 
[tests : nosetests] ======================================================================
[tests : nosetests] ERROR: Failure: AttributeError (module 'sqlalchemy' has no attribute '__all__')
[tests : nosetests] ----------------------------------------------------------------------
[tests : nosetests] Traceback (most recent call last):
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/failure.py", line 39, in runTest
[tests : nosetests]     raise self.exc_val.with_traceback(self.tb)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/loader.py", line 417, in loadTestsFromName
[tests : nosetests]     module = self.importer.importFromPath(
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/importer.py", line 47, in importFromPath
[tests : nosetests]     return self.importFromDir(dir_path, fqname)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/importer.py", line 94, in importFromDir
[tests : nosetests]     mod = load_module(part_fqname, fh, filename, desc)
[tests : nosetests]   File "/usr/local/lib/python3.9/imp.py", line 234, in load_module
[tests : nosetests]     return load_source(name, filename, file)
[tests : nosetests]   File "/usr/local/lib/python3.9/imp.py", line 171, in load_source
[tests : nosetests]     module = _load(spec)
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 711, in _load
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
[tests : nosetests]   File "<frozen importlib._bootstrap_external>", line 850, in exec_module
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
[tests : nosetests]   File "/workspace/source/tests/test_models.py", line 8, in <module>
[tests : nosetests]     from service import app
[tests : nosetests]   File "/workspace/source/service/__init__.py", line 22, in <module>
[tests : nosetests]     from service import routes, models  # noqa: F401 E402
[tests : nosetests]   File "/workspace/source/service/routes.py", line 8, in <module>
[tests : nosetests]     from service.models import Account
[tests : nosetests]   File "/workspace/source/service/models.py", line 13, in <module>
[tests : nosetests]     db = SQLAlchemy()
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 758, in __init__
[tests : nosetests]     _include_sqlalchemy(self, query_class)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 112, in _include_sqlalchemy
[tests : nosetests]     for key in module.__all__:
[tests : nosetests] AttributeError: module 'sqlalchemy' has no attribute '__all__'
[tests : nosetests] 
[tests : nosetests] ======================================================================
[tests : nosetests] ERROR: Failure: AttributeError (module 'sqlalchemy' has no attribute '__all__')
[tests : nosetests] ----------------------------------------------------------------------
[tests : nosetests] Traceback (most recent call last):
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/failure.py", line 39, in runTest
[tests : nosetests]     raise self.exc_val.with_traceback(self.tb)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/loader.py", line 417, in loadTestsFromName
[tests : nosetests]     module = self.importer.importFromPath(
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/importer.py", line 47, in importFromPath
[tests : nosetests]     return self.importFromDir(dir_path, fqname)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/nose/importer.py", line 94, in importFromDir
[tests : nosetests]     mod = load_module(part_fqname, fh, filename, desc)
[tests : nosetests]   File "/usr/local/lib/python3.9/imp.py", line 234, in load_module
[tests : nosetests]     return load_source(name, filename, file)
[tests : nosetests]   File "/usr/local/lib/python3.9/imp.py", line 171, in load_source
[tests : nosetests]     module = _load(spec)
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 711, in _load
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
[tests : nosetests]   File "<frozen importlib._bootstrap_external>", line 850, in exec_module
[tests : nosetests]   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
[tests : nosetests]   File "/workspace/source/tests/test_routes.py", line 11, in <module>
[tests : nosetests]     from tests.factories import AccountFactory
[tests : nosetests]   File "/workspace/source/tests/factories.py", line 7, in <module>
[tests : nosetests]     from service.models import Account
[tests : nosetests]   File "/workspace/source/service/__init__.py", line 22, in <module>
[tests : nosetests]     from service import routes, models  # noqa: F401 E402
[tests : nosetests]   File "/workspace/source/service/routes.py", line 8, in <module>
[tests : nosetests]     from service.models import Account
[tests : nosetests]   File "/workspace/source/service/models.py", line 13, in <module>
[tests : nosetests]     db = SQLAlchemy()
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 758, in __init__
[tests : nosetests]     _include_sqlalchemy(self, query_class)
[tests : nosetests]   File "/usr/local/lib/python3.9/site-packages/flask_sqlalchemy/__init__.py", line 112, in _include_sqlalchemy
[tests : nosetests]     for key in module.__all__:
[tests : nosetests] AttributeError: module 'sqlalchemy' has no attribute '__all__'
[tests : nosetests] 
[tests : nosetests] Name                             Stmts   Miss  Cover   Missing
[tests : nosetests] --------------------------------------------------------------
[tests : nosetests] service/common/__init__.py           0      0   100%
[tests : nosetests] service/common/log_handlers.py      10      8    20%   12-22
[tests : nosetests] service/config.py                   11      5    55%   12-16
[tests : nosetests] --------------------------------------------------------------
[tests : nosetests] TOTAL                               21     13    38%
[tests : nosetests] ----------------------------------------------------------------------
[tests : nosetests] Ran 4 tests in 1.112s
[tests : nosetests] 
[tests : nosetests] FAILED (errors=4)
[tests : nosetests] 

[lint : flake8] Collecting Werkzeug==2.1.2
[lint : flake8]   Downloading Werkzeug-2.1.2-py3-none-any.whl (224 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 224.9/224.9 KB 47.6 MB/s eta 0:00:00
[lint : flake8] Collecting Flask==2.1.2
[lint : flake8]   Downloading Flask-2.1.2-py3-none-any.whl (95 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 95.2/95.2 KB 30.3 MB/s eta 0:00:00
[lint : flake8] Collecting Flask-SQLAlchemy==2.5.1
[lint : flake8]   Downloading Flask_SQLAlchemy-2.5.1-py2.py3-none-any.whl (17 kB)
[lint : flake8] Collecting psycopg2-binary==2.9.3
[lint : flake8]   Downloading psycopg2_binary-2.9.3-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.0 MB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 3.0/3.0 MB 39.1 MB/s eta 0:00:00
[lint : flake8] Collecting python-dotenv==0.20.0
[lint : flake8]   Downloading python_dotenv-0.20.0-py3-none-any.whl (17 kB)
[lint : flake8] Collecting flask-talisman
[lint : flake8]   Downloading flask_talisman-1.0.0-py2.py3-none-any.whl (18 kB)
[lint : flake8] Collecting flask-cors
[lint : flake8]   Downloading Flask_Cors-3.0.10-py2.py3-none-any.whl (14 kB)
[lint : flake8] Collecting gunicorn==20.1.0
[lint : flake8]   Downloading gunicorn-20.1.0-py3-none-any.whl (79 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 79.5/79.5 KB 41.8 MB/s eta 0:00:00
[lint : flake8] Collecting honcho==1.1.0
[lint : flake8]   Downloading honcho-1.1.0-py2.py3-none-any.whl (21 kB)
[lint : flake8] Collecting pylint==2.14.0
[lint : flake8]   Downloading pylint-2.14.0-py3-none-any.whl (485 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 485.0/485.0 KB 70.3 MB/s eta 0:00:00
[lint : flake8] Collecting flake8==4.0.1
[lint : flake8]   Downloading flake8-4.0.1-py2.py3-none-any.whl (64 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 64.1/64.1 KB 30.7 MB/s eta 0:00:00
[lint : flake8] Collecting black==22.3.0
[lint : flake8]   Downloading black-22.3.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.5 MB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 1.5/1.5 MB 77.2 MB/s eta 0:00:00
[lint : flake8] Collecting nose==1.3.7
[lint : flake8]   Downloading nose-1.3.7-py3-none-any.whl (154 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 154.7/154.7 KB 42.3 MB/s eta 0:00:00
[lint : flake8] Collecting pinocchio==0.4.3
[lint : flake8]   Downloading pinocchio-0.4.3-py3-none-any.whl (12 kB)
[lint : flake8] Collecting factory-boy==2.12.0
[lint : flake8]   Downloading factory_boy-2.12.0-py2.py3-none-any.whl (36 kB)
[lint : flake8] Collecting coverage==6.3.2
[lint : flake8]   Downloading coverage-6.3.2-cp39-cp39-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (210 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 210.7/210.7 KB 74.3 MB/s eta 0:00:00
[lint : flake8] Collecting codecov==2.1.12
[lint : flake8]   Downloading codecov-2.1.12-py2.py3-none-any.whl (16 kB)
[lint : flake8] Collecting httpie==3.2.1
[lint : flake8]   Downloading httpie-3.2.1-py3-none-any.whl (124 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 125.0/125.0 KB 38.0 MB/s eta 0:00:00
[lint : flake8] Collecting importlib-metadata>=3.6.0
[lint : flake8]   Downloading importlib_metadata-6.0.0-py3-none-any.whl (21 kB)
[lint : flake8] Collecting itsdangerous>=2.0
[lint : flake8]   Downloading itsdangerous-2.1.2-py3-none-any.whl (15 kB)
[lint : flake8] Collecting Jinja2>=3.0
[lint : flake8]   Downloading Jinja2-3.1.2-py3-none-any.whl (133 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 133.1/133.1 KB 35.2 MB/s eta 0:00:00
[lint : flake8] Collecting click>=8.0
[lint : flake8]   Downloading click-8.1.3-py3-none-any.whl (96 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 96.6/96.6 KB 33.1 MB/s eta 0:00:00
[lint : flake8] Collecting SQLAlchemy>=0.8.0
[lint : flake8]   Downloading SQLAlchemy-2.0.0-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (2.8 MB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 2.8/2.8 MB 103.3 MB/s eta 0:00:00
[lint : flake8] Requirement already satisfied: setuptools>=3.0 in /usr/local/lib/python3.9/site-packages (from gunicorn==20.1.0->-r requirements.txt (line 13)) (58.1.0)
[lint : flake8] Collecting typing-extensions>=3.10.0
[lint : flake8]   Downloading typing_extensions-4.4.0-py3-none-any.whl (26 kB)
[lint : flake8] Collecting platformdirs>=2.2.0
[lint : flake8]   Downloading platformdirs-2.6.2-py3-none-any.whl (14 kB)
[lint : flake8] Collecting tomli>=1.1.0
[lint : flake8]   Downloading tomli-2.0.1-py3-none-any.whl (12 kB)
[lint : flake8] Collecting astroid<=2.12.0-dev0,>=2.11.5
[lint : flake8]   Downloading astroid-2.11.7-py3-none-any.whl (251 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 251.2/251.2 KB 74.9 MB/s eta 0:00:00
[lint : flake8] Collecting dill>=0.2
[lint : flake8]   Downloading dill-0.3.6-py3-none-any.whl (110 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 110.5/110.5 KB 2.1 MB/s eta 0:00:00
[lint : flake8] Collecting isort<6,>=4.2.5
[lint : flake8]   Downloading isort-5.12.0-py3-none-any.whl (91 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 91.2/91.2 KB 42.6 MB/s eta 0:00:00
[lint : flake8] Collecting tomlkit>=0.10.1
[lint : flake8]   Downloading tomlkit-0.11.6-py3-none-any.whl (35 kB)
[lint : flake8] Collecting mccabe<0.8,>=0.6
[lint : flake8]   Downloading mccabe-0.7.0-py2.py3-none-any.whl (7.3 kB)
[lint : flake8] Collecting pycodestyle<2.9.0,>=2.8.0
[lint : flake8]   Downloading pycodestyle-2.8.0-py2.py3-none-any.whl (42 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 42.1/42.1 KB 23.0 MB/s eta 0:00:00
[lint : flake8] Collecting pyflakes<2.5.0,>=2.4.0
[lint : flake8]   Downloading pyflakes-2.4.0-py2.py3-none-any.whl (69 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 69.7/69.7 KB 27.0 MB/s eta 0:00:00
[lint : flake8] Collecting mccabe<0.8,>=0.6
[lint : flake8]   Downloading mccabe-0.6.1-py2.py3-none-any.whl (8.6 kB)
[lint : flake8] Collecting pathspec>=0.9.0
[lint : flake8]   Downloading pathspec-0.11.0-py3-none-any.whl (29 kB)
[lint : flake8] Collecting mypy-extensions>=0.4.3
[lint : flake8]   Downloading mypy_extensions-0.4.3-py2.py3-none-any.whl (4.5 kB)
[lint : flake8] Collecting colorama
[lint : flake8]   Downloading colorama-0.4.6-py2.py3-none-any.whl (25 kB)
[lint : flake8] Collecting Faker>=0.7.0
[lint : flake8]   Downloading Faker-16.6.1-py3-none-any.whl (1.7 MB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 1.7/1.7 MB 103.4 MB/s eta 0:00:00
[lint : flake8] Collecting requests>=2.7.9
[lint : flake8]   Downloading requests-2.28.2-py3-none-any.whl (62 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 62.8/62.8 KB 32.6 MB/s eta 0:00:00
[lint : flake8] Requirement already satisfied: pip in /usr/local/lib/python3.9/site-packages (from httpie==3.2.1->-r requirements.txt (line 32)) (22.0.4)
[lint : flake8] Collecting multidict>=4.7.0
[lint : flake8]   Downloading multidict-6.0.4-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (114 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 114.2/114.2 KB 53.5 MB/s eta 0:00:00
[lint : flake8] Collecting requests-toolbelt>=0.9.1
[lint : flake8]   Downloading requests_toolbelt-0.10.1-py2.py3-none-any.whl (54 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 54.5/54.5 KB 29.3 MB/s eta 0:00:00
[lint : flake8] Collecting defusedxml>=0.6.0
[lint : flake8]   Downloading defusedxml-0.7.1-py2.py3-none-any.whl (25 kB)
[lint : flake8] Collecting charset-normalizer>=2.0.0
[lint : flake8]   Downloading charset_normalizer-3.0.1-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (198 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 198.8/198.8 KB 71.0 MB/s eta 0:00:00
[lint : flake8] Collecting Pygments>=2.5.2
[lint : flake8]   Downloading Pygments-2.14.0-py3-none-any.whl (1.1 MB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 1.1/1.1 MB 101.7 MB/s eta 0:00:00
[lint : flake8] Collecting rich>=9.10.0
[lint : flake8]   Downloading rich-13.3.1-py3-none-any.whl (239 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 239.0/239.0 KB 78.0 MB/s eta 0:00:00
[lint : flake8] Collecting Six
[lint : flake8]   Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
[lint : flake8] Collecting wrapt<2,>=1.11
[lint : flake8]   Downloading wrapt-1.14.1-cp39-cp39-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (77 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 77.8/77.8 KB 33.0 MB/s eta 0:00:00
[lint : flake8] Collecting lazy-object-proxy>=1.4.0
[lint : flake8]   Downloading lazy_object_proxy-1.9.0-cp39-cp39-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (62 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 62.1/62.1 KB 31.4 MB/s eta 0:00:00
[lint : flake8] Collecting python-dateutil>=2.4
[lint : flake8]   Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 247.7/247.7 KB 4.6 MB/s eta 0:00:00
[lint : flake8] Collecting zipp>=0.5
[lint : flake8]   Downloading zipp-3.12.0-py3-none-any.whl (6.6 kB)
[lint : flake8] Collecting MarkupSafe>=2.0
[lint : flake8]   Downloading MarkupSafe-2.1.2-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (25 kB)
[lint : flake8] Collecting certifi>=2017.4.17
[lint : flake8]   Downloading certifi-2022.12.7-py3-none-any.whl (155 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 155.3/155.3 KB 53.8 MB/s eta 0:00:00
[lint : flake8] Collecting urllib3<1.27,>=1.21.1
[lint : flake8]   Downloading urllib3-1.26.14-py2.py3-none-any.whl (140 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 140.6/140.6 KB 53.9 MB/s eta 0:00:00
[lint : flake8] Collecting idna<4,>=2.5
[lint : flake8]   Downloading idna-3.4-py3-none-any.whl (61 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 61.5/61.5 KB 31.3 MB/s eta 0:00:00
[lint : flake8] Collecting PySocks!=1.5.7,>=1.5.6
[lint : flake8]   Downloading PySocks-1.7.1-py3-none-any.whl (16 kB)
[lint : flake8] Collecting markdown-it-py<3.0.0,>=2.1.0
[lint : flake8]   Downloading markdown_it_py-2.1.0-py3-none-any.whl (84 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 84.5/84.5 KB 28.6 MB/s eta 0:00:00
[lint : flake8] Collecting greenlet!=0.4.17
[lint : flake8]   Downloading greenlet-2.0.2-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (610 kB)
[lint : flake8]      ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ‚îÅ 610.9/610.9 KB 91.6 MB/s eta 0:00:00
[lint : flake8] Collecting mdurl~=0.1
[lint : flake8]   Downloading mdurl-0.1.2-py3-none-any.whl (10.0 kB)
[lint : flake8] Installing collected packages: nose, mypy-extensions, mccabe, honcho, flask-talisman, charset-normalizer, zipp, wrapt, Werkzeug, urllib3, typing-extensions, tomlkit, tomli, Six, python-dotenv, PySocks, Pygments, pyflakes, pycodestyle, psycopg2-binary, platformdirs, pathspec, multidict, mdurl, MarkupSafe, lazy-object-proxy, itsdangerous, isort, idna, gunicorn, greenlet, dill, defusedxml, coverage, colorama, click, certifi, SQLAlchemy, requests, python-dateutil, pinocchio, markdown-it-py, Jinja2, importlib-metadata, flake8, black, astroid, rich, requests-toolbelt, pylint, Flask, Faker, codecov, httpie, Flask-SQLAlchemy, flask-cors, factory-boy
[lint : flake8] Successfully installed Faker-16.6.1 Flask-2.1.2 Flask-SQLAlchemy-2.5.1 Jinja2-3.1.2 MarkupSafe-2.1.2 PySocks-1.7.1 Pygments-2.14.0 SQLAlchemy-2.0.0 Six-1.16.0 Werkzeug-2.1.2 astroid-2.11.7 black-22.3.0 certifi-2022.12.7 charset-normalizer-3.0.1 click-8.1.3 codecov-2.1.12 colorama-0.4.6 coverage-6.3.2 defusedxml-0.7.1 dill-0.3.6 factory-boy-2.12.0 flake8-4.0.1 flask-cors-3.0.10 flask-talisman-1.0.0 greenlet-2.0.2 gunicorn-20.1.0 honcho-1.1.0 httpie-3.2.1 idna-3.4 importlib-metadata-6.0.0 isort-5.12.0 itsdangerous-2.1.2 lazy-object-proxy-1.9.0 markdown-it-py-2.1.0 mccabe-0.6.1 mdurl-0.1.2 multidict-6.0.4 mypy-extensions-0.4.3 nose-1.3.7 pathspec-0.11.0 pinocchio-0.4.3 platformdirs-2.6.2 psycopg2-binary-2.9.3 pycodestyle-2.8.0 pyflakes-2.4.0 pylint-2.14.0 python-dateutil-2.8.2 python-dotenv-0.20.0 requests-2.28.2 requests-toolbelt-0.10.1 rich-13.3.1 tomli-2.0.1 tomlkit-0.11.6 typing-extensions-4.4.0 urllib3-1.26.14 wrapt-1.14.1 zipp-3.12.0
[lint : flake8] WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
[lint : flake8] WARNING: You are using pip version 22.0.4; however, version 22.3.1 is available.
[lint : flake8] You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.
[lint : flake8] ./service/routes.py:63:1: E302 expected 2 blank lines, found 1
[lint : flake8] ./service/routes.py:82:1: E303 too many blank lines (6)
[lint : flake8] ./service/routes.py:85:1: E302 expected 2 blank lines, found 6
[lint : flake8] ./service/routes.py:92:1: W293 blank line contains whitespace
[lint : flake8] ./service/routes.py:96:1: W293 blank line contains whitespace
[lint : flake8] ./service/routes.py:98:1: W293 blank line contains whitespace
[lint : flake8] ./service/routes.py:102:1: E303 too many blank lines (4)
[lint : flake8] ./service/routes.py:105:1: E302 expected 2 blank lines, found 4
[lint : flake8] ./service/routes.py:113:1: W293 blank line contains whitespace
[lint : flake8] ./service/routes.py:115:20: W291 trailing whitespace
[lint : flake8] ./service/routes.py:117:1: W293 blank line contains whitespace
[lint : flake8] ./service/routes.py:134:1: W293 blank line contains whitespace
[lint : flake8] ./service/routes.py:137:13: E117 over-indented
[lint : flake8] ./tests/test_routes.py:141:1: W293 blank line contains whitespace
[lint : flake8] ./tests/test_routes.py:142:5: E303 too many blank lines (2)
[lint : flake8] ./tests/test_routes.py:149:1: W293 blank line contains whitespace
[lint : flake8] ./tests/test_routes.py:170:1: W293 blank line contains whitespace
[lint : flake8] ./tests/test_routes.py:175:1: W293 blank line contains whitespace
[lint : flake8] ./tests/test_routes.py:178:58: E251 unexpected spaces around keyword / parameter equals
[lint : flake8] ./tests/test_routes.py:178:60: E251 unexpected spaces around keyword / parameter equals
[lint : flake8] ./tests/test_routes.py:178:75: W291 trailing whitespace
[lint : flake8] ./tests/test_routes.py:190:1: W293 blank line contains whitespace
[lint : flake8] ./tests/test_routes.py:193:58: E251 unexpected spaces around keyword / parameter equals
[lint : flake8] ./tests/test_routes.py:193:60: E251 unexpected spaces around keyword / parameter equals
[lint : flake8] ./tests/test_routes.py:193:75: W291 trailing whitespace
[lint : flake8] ./tests/test_routes.py:199:1: W293 blank line contains whitespace
[lint : flake8] ./tests/test_routes.py:201:1: W391 blank line at end of file
[lint : flake8] 1     E117 over-indented
[lint : flake8] 4     E251 unexpected spaces around keyword / parameter equals
[lint : flake8] 3     E302 expected 2 blank lines, found 1
[lint : flake8] 3     E303 too many blank lines (6)
[lint : flake8] 3     W291 trailing whitespace
[lint : flake8] 12    W293 blank line contains whitespace
[lint : flake8] 1     W391 blank line at end of file
[lint : flake8] 27

